YANG/ccamp/layer1-types/ietf-layer1-types.yang

```diff
@@ -1101,38 +1117,42 @@ module ietf-layer1-types {
     description "OTN Label";
     reference
       "RFC7139, section 6: GMPLS Signaling Extensions for Control of
-       Evolving G.709 Optical Transport Networks"; 
-    leaf otn-tpn {
-      type otn-tpn;
+       Evolving G.709 Optical Transport Networks";
+    container otn {
       description
-        "Tributary Port Number.";
-      reference
-        "RFC7139: GMPLS Signaling Extensions for Control of Evolving
-         G.709 Optical Transport Networks.";
-    }
-    leaf tsg {
-      type identityref {
-        base tributary-slot-granularity;
+        "Label hop for OTN.";
+      leaf otn-tpn {
```

Comment:
If we add an "otn" container then here "otn-" prefix is really unnecessary. Suggest changing it to just "tpn"

Reply:
I agree on the comment above, please delete "otn" up front to leaf "otn-tpn"

Reply:
Addressed in https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/104/commits/b80b3a8f5fbf26efc4b237f06b859a03e133b297